<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maindialog</class>
 <widget class="QDialog" name="Maindialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maindialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>20</y>
     <width>241</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Source.qrc">:/C:/Users/00/Pictures/AA/txfb.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_page">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>9</y>
       <width>311</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_password_login">
        <property name="text">
         <string>000000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_tel_login">
        <property name="text">
         <string>18845621888</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pb_clear_login">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>88</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_login_login">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>88</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>214</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_name_register"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_password_register"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_confirm_register"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_tel_register"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>密码确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>88</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_register_register">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>88</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_tel_login</tabstop>
  <tabstop>le_password_login</tabstop>
  <tabstop>pb_login_login</tabstop>
  <tabstop>pb_clear_login</tabstop>
  <tabstop>le_name_register</tabstop>
  <tabstop>le_tel_register</tabstop>
  <tabstop>le_password_register</tabstop>
  <tabstop>le_confirm_register</tabstop>
  <tabstop>pb_register_register</tabstop>
  <tabstop>pb_clear_register</tabstop>
  <tabstop>tw_page</tabstop>
 </tabstops>
 <resources>
  <include location="Source.qrc"/>
 </resources>
 <connections/>
</ui>
